packages/wrangler/src/pages/dev.tsx

```diff
@@ -40,6 +40,27 @@ export function Options(yargs: Argv) {
 				default: true,
 				description: "Run on my machine",
 			},
+			"compatibility-date": {
+				describe: "Date to use for compatibility checks",
+				type: "string",
+			},
+			"compatibility-flags": {
+				describe: "Flags to use for compatibility checks",
+				alias: "compatibility-flag",
```

Comment:
@GregBrimble why the alias? won't this be confusing?

Reply:
https://github.com/cloudflare/wrangler2/blob/main/packages/wrangler/src/dev.tsx#L117-L123

Just to make it line up with what `wrangler dev` does. Both work :)

Reply:
👍